css/impact.css

```diff
@@ -230,8 +253,25 @@ Program Gallery
 	.program-gallery {
 		padding: var(--spacing-med) var(--spacing-sm);
 	}
+
+	/* gradient graphic adjustments */
+	.programs::before {
```

Comment:
having to do a lot of experimenting w magic numbers - background looks more congruent for each breakpoint.